### PR TITLE
Popup Window

### PR DESCRIPTION
**Pull request by Kevin-Mena**
**Portfolio Website:Pop up Window**
in this Milestone I;
-Added pop up window

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+              <img src="img/seelive.png" alt="visit the website"/>
+            </a>
+            <a href="${main[i].source}" class="btn">
+              see source
+              <img src="img/seesource.png" alt=" see the sourcecode"/>
+            </a>
+          </div>
+        </div>
+        <img src="img/cancel.png" alt="To close panel" id="cancel-port"/>
+      </div>`;
```
Comment (Shedrack-Sunday) [inline script.js]: - [ ] Firstly, I must say that your project is awesome overall 👏  You really took time to give a great design. However, there's a slight variation between the design of your skills and source links to your project, with what's presented in the figma design. The source button that directs to the `live` version and project `source code` does not show up in the mobile view. It's visible on the desktop view, and perfectly set, and I strongly suggest you make a similar design on the mobile. You'd agree with me that this gives a clear path for web accessibility for mobile users and gives your potential viewers paths to authenticate your project from the source and have a clear web view of what it looks like.

- [ ] Note that at the initial load, the buttons are present, but they don't render the view is rendered. kindly check the view below 💯 

# Your present view:
image

# View expected from figma design :
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Kevin-Mena     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
✅ Beautiful design 👏 


Required Changes ♻️

- [x] Your Project looks decent, and you really need a great job. However, there seems to be a little be of adjustment needed on the mobile pop-up, to make your whole code stand out as it is already. Kindly check the comments for a detailed review 👍 


_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
 


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: script.js [hunk 2]
```diff
@@ -1,15 +1,166 @@
...
+    for (let i = 0; i < array.length; i += 1) {
+      if (i === v) {
+        display(array, i);
+        const view = document.querySelector('.popup');
+        view.classList.add('show');
+
+        const canc = view.querySelector('#cancel-port');
+        canc.addEventListener('click', () => {
+          extra.removeChild(view);
+        });
```
Comment (Reem-lab) [inline script.js:162]: - [ OPTIONAL ] Great job on making the required things but I highly recommend to add hover on the buttons for the `source link` and `live version` so this is button interaction that's improve from user experience (UX) so kindly add it 🔥💯😉🙏🏻🙏🏻

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Followed GitHub flow ✅
- Great working on the pop up functionality✅
- Professional README ✅
- Descriptive PR 💯💯


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**